lms/templates/courseware/courseware.html

```diff
@@ -14,6 +15,11 @@
 </%def>
 
 <%block name="bodyclass">view-in-course view-courseware courseware ${course.css_class or ''}</%block>
+
+<%block name="js_overrides">
```

Comment:
Agreed.  This will cause issues for clients that use courseware-chromeless: mobile and LTI provider.